packages/mui-material/src/TablePaginationActions/TablePaginationActions.test.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { createRenderer } from '@mui/internal-test-utils';
+import TablePaginationActions, {
+  tablePaginationActionsClasses as classes,
+} from '@mui/material/TablePaginationActions';
+import describeConformance from '../../test/describeConformance';
+
+describe('<TablePaginationActions />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(
+    <TablePaginationActions
+      getItemAriaLabel={(type) => {
+        if (type === 'first') {
+          return 'first';
+        }
+        if (type === 'last') {
+          return 'last';
+        }
+        if (type === 'next') {
+          return 'next';
+        }
+        return 'previous';
+      }}
+    />,
+    () => ({
+      inheritComponent: 'div',
+      render,
+      classes,
+      muiName: 'MuiTablePaginationActions',
+      refInstanceof: window.HTMLDivElement,
+      skip: ['componentsProp', 'componentProp', 'themeVariants'],
```

Comment:
Removed `classesRoot`, `themeDefaultProps`, and `themeStyleOverrides`. This test serves as proof that applying the root class and setting props from the theme are working correctly.